src/routes/accommodationRoute.js

```diff
@@ -17,21 +17,26 @@ const fUpload = fileUpload({
 });
 
 const {
-  isSupplierAccommodation, checkForImages, checkForImagesUpdate
+  isSupplierAccommodation, checkForImages, checkForImagesUpdate, accommodationExists
 } = AccommodationMiddleware;
 const { checkToken } = UserMiddleware;
 
 app.patch('/:id', fUpload, checkToken, RoleMiddleware.checkHostOrTAdmin, isSupplierAccommodation, checkForImagesUpdate, valid.editAccommodation, AccommodationsController.editAccommodation);
 app.delete('/:id', checkToken, RoleMiddleware.checkHost, isSupplierAccommodation, AccommodationsController.deleteAccommodation);
 app.post('/', fUpload, checkToken, RoleMiddleware.checkHostOrTAdmin, valid.accommodation, checkForImages, AccommodationsController.addAccommodation);
 app.get('/', AccommodationsController.getAccommodations);
-app.get('/:accommodationId/rating', valid.getReviewvalidation, BookingsController.getRating);
+app.get('/:accommodationId/rating', valid.accommodationId, BookingsController.getRating);
 app.get('/search', AccommodationsController.getAccommodationsByFilter);
+app.get('/bookmarked', checkToken, AccommodationsController.getBookmarkedAccommodations);
 app.get('/:accommodationId', AccommodationsController.getAccommodationById);
 app.post('/:accommodationId/like', [
   checkToken,
   LikingsController.addLikeAccommdation
 ]);
-
-
+app.post('/:accommodationId/bookmark', [checkToken, valid.accommodationId,
+  accommodationExists,
+  AccommodationsController.bookmarkAccommodation]);
+app.delete('/:accommodationId/bookmark', [checkToken, valid.accommodationId,
+  accommodationExists,
+  AccommodationsController.undoBookmark]);
```

Comment:
We have a format that we've been using, yeah? Why this new array format for middlewares?